pyanaconda/modules/subscription/installation.py

```diff
@@ -199,6 +216,19 @@ def _transfer_file(self, target_path, target_name):
             msg = "{} ({}) is missing".format(target_name, self.RHSM_REPO_FILE_PATH)
             raise SubscriptionTokenTransferError(msg)
 
+    def _transfer_folder(self, target_path, target_name):
```

Comment:
Here as well. :)

Reply:
I decided to keep this one here because the reusable moving logic is on `copy_folder`. this method is just wrapping it and adding the proper error.